mapentity/templates/mapentity/mapentity_filter.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n static %}
+
+{% block mainfilter %}
+<form id="mainfilter" action="{{ model.get_datatablelist_url }}" filter-url="{{ filter_url }}">
+    <div class="left form-group">
+        {{ filter.form.as_p }}
+    </div>
+    <div class="right form-group"></div>
+    <div class="actions">
+        <input id="reset" type="reset" class="btn btn-light"/>
+        <a id="filter" class="btn btn-primary"><i class="bi bi-search"></i> {% trans "Filter" %}</a>
```

Comment:
It should be great to transform this `<a>` tag by a `<button type="submit">` and then replace the following code in `mapentity/static/mapentity/mapentity.filter.js`

```js
$("#filter").click(function(e) {
     mapsync._onFormSubmit(e);
});
```

by

```js
$('#mainfilter').submit(function(e) {
     e.preventDefault();
     mapsync._onFormSubmit(e.submitter);
});
```